lib/jira/resource/user.rb

```diff
@@ -13,7 +14,7 @@ def self.singular_path(client, key, prefix = '/')
       # This method is a bit of a hack. There is no way to get a list of all users in Jira, so we concatenate all of the project IDs
       # and use the assignable users endpoint to get a list of users that can be assigned to all projects.
       def self.all(client)
-        project_list = client.Project.all.map(&:key).join(',')
+        project_list = client.Project.all.map(&:key).take(SMALLER_SAMPLE_SIZE).join(',')
```

Comment:
So does this mean we might be excluding some users from the returned results because we're omitting projects the user might be on? If so, should we consider batching requests for users on projects and then finding all of the unique users after we've fetched all of the users on all of the projects?

Reply:
@amandamholl Yes! I like that a lot better, because even with the original method we could still be missing users. Thanks, I will work on that.